ucm2/MediaTek/mt8365-evk/HiFi.conf

```diff
@@ -0,0 +1,162 @@
+SectionVerb {
+        EnableSequence [
+		disdevall ""
+	]
+}
+
+SectionDevice."HDMI" {
+        Comment "Hdmi output"
+
+        Value {
+                PlaybackPriority 250
+                PlaybackChannels 2
+                PlaybackPCM "hw:${CardId},1"
+        }
+
+	EnableSequence [
+		cset "name='O00 I07 Switch' on"
+		cset "name='O01 I08 Switch' on"
+	]
+
+	DisableSequence [
+		cset "name='O00 I07 Switch' off"
+		cset "name='O01 I08 Switch' off"
+	]
+}
+
+SectionDevice."Headphones" {
+	Comment "Headset speakers"
+
+	ConflictingDevice [
+		"Speaker"
+        ]
+
+	Value {
+		PlaybackPriority 300
+		PlaybackChannels 2
+		PlaybackPCM "hw:${CardId},0"
+		PlaybackVolume "name='Headset_PGAL_GAIN'"
+        }
+
+	EnableSequence [
+		cset "name='Audio_Amp_L_Switch' on"
+		cset "name='Audio_Amp_R_Switch' on"
+		cset "name='Headset_PGAL_GAIN' 1"
+		cset "name='Speaker_Amp_Switch' off"
+        ]
+
+	DisableSequence [
+		cset "name='Audio_Amp_L_Switch' off"
+		cset "name='Audio_Amp_R_Switch' off"
+		cset "name='Speaker_Amp_Switch' on"
+	]
+}
+
+SectionDevice."Speaker" {
+	Comment "Line-out Jack "
+
+	ConflictingDevice [
+		"Headphones"
+        ]
+
+	Value {
+		PlaybackPriority 300
+		PlaybackChannels 2
+		PlaybackPCM "hw:${CardId},0"
+		PlaybackVolume "name='Lineout_PGAL_GAIN'"
+        }
+
+	EnableSequence [
+		cset "name='Audio_Amp_L_Switch' off"
+		cset "name='Audio_Amp_R_Switch' off"
+		cset "name='Lineout_PGAL_GAIN' 0"
+		cset "name='Speaker_Amp_Switch' on"
+	]
+
+	DisableSequence [
+		cset "name='Audio_Amp_L_Switch' on"
+		cset "name='Audio_Amp_R_Switch' on"
+		cset "name='Speaker_Amp_Switch' off"
+        ]
+}
+
+SectionDevice."Mic1" {
+	Comment "Headset microphone"
+
+	ConflictingDevice [
+		"Mic2"
+	]
+
+	Value {
+                CapturePriority 300
+                CaptureChannels 1
+                CapturePCM "hw:${CardId},2"
+        }
+
+	EnableSequence [
+		cset "name='Audio_Amp_L_Switch' on"
```

Comment:
Why you change output paths for the input device ? It seems like a conflict.

Reply:
To make sure that the sound's output is "Headset Out". 
I removed these commands from Mic1, Mic2 and Mic3 and retested. Still get no sound with PA. 

I can see that the audio is connected to right sink  ( alsa_output.platform-sound.HiFi__hw_mt8365evk_0__sink) but still can't understand why I get no sound...
```
i350-evk:~$ paplay -vvv music.wav 
Opening a playback stream with sample specification 's32le 2ch 16000Hz' and channel map 'front-left,front-right'.
Connection established.
Stream successfully created.
Buffer metrics: maxlength=4194304, tlength=256000, prebuf=253448, minreq=2560
Using sample spec 's32le 2ch 16000Hz', channel map 'front-left,front-right'.
Connected to device alsa_output.platform-sound.HiFi__hw_mt8365evk_0__sink (index: 0, suspended: no). 
```